spec/system/task_spec.rb

```diff
@@ -1,27 +1,29 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task) { create(:task, project: project) }
```

Comment:
`association`をFactoryに記載するとprojectの指定を省略できますね。
```suggestion
  let(:task) { create(:task) }
```

Reply:
64cd8d52c5596588c94ab220081876b51bdca2a7 でassociationを追加、リファクタリングしました